### ISSUE TITLE
powelevel10k configuration wizard works at each opening (even though I configured it)

### ISSUE THREAD

**Issue by barisozmen** [romkatv/powerlevel10k#176]
At each opening of my terminal (iTerm2), I encounter with the screen below. 
![image](https://user-images.githubusercontent.com/14996155/63241092-4635c780-c207-11e9-8e31-d810cdc17773.png)

I'm answering all questions and configure it, but it asks again when I open the terminal another time. 

When I finish configuring, I see below screen, which states "Powerlevel10k config file already exists" which should be true because I have already configured.
![image](https://user-images.githubusercontent.com/14996155/63241182-92810780-c207-11e9-81a4-6cd820e077f2.png)

When I say "yes", I am getting following screen.
![image](https://user-images.githubusercontent.com/14996155/63241210-b2183000-c207-11e9-8755-ab5bd11e7b7d.png)

If I say "quit" at the beginning of configuration, I am getting following screen with `echo 'POWERLEVEL9K_DISABLE_CONFIGURATION_WIZARD=true' >>! ~/.zshrc` suggestion.
![image](https://user-images.githubusercontent.com/14996155/63241272-f9062580-c207-11e9-9900-edc13da2583a.png)

However, even if I apply the suggestion, it doesn't work. 

I also tried doing `echo 'POWERLEVEL10K_DISABLE_CONFIGURATION_WIZARD=true' >>! ~/.zshrc`. But it didn't work either.

I have to enter same configurations each time I open a new window.

Do you have any idea how can I solve this?

Thanks!

**Comment by romkatv** [comment]
> If I say "quit" at the beginning of configuration, I am getting following screen with `echo 'POWERLEVEL9K_DISABLE_CONFIGURATION_WIZARD=true' >>! ~/.zshrc` suggestion.

What does your prompt look like in this case? The screenshot you posted is cut-off right before the prompt line. Can you show that line?

> ![image](https://user-images.githubusercontent.com/14996155/63241272-f9062580-c207-11e9-9900-edc13da2583a.png)
> 
> However, even if I apply the suggestion, it doesn't work.

My guess is that your `~/.zshrc` either doesn't get sourced or it exits early without reaching the end.

How did you install powerlevel10k?

Could you attach your `~/.zshrc`.

> I also tried doing `echo 'POWERLEVEL10K_DISABLE_CONFIGURATION_WIZARD=true' >>! ~/.zshrc`. But it didn't work either.

This is expected. `POWERLEVEL10K_DISABLE_CONFIGURATION_WIZARD` has no special meaning.

**Comment by barisozmen** [comment]
I uninstalled and re-installed zsh and powerlevel10k, and my problem solved.

I followed these steps to solve:
1. Install zsh (following https://github.com/robbyrussell/oh-my-zsh#via-curl)
2. Install powerlevel10k (following https://github.com/romkatv/powerlevel10k#oh-my-zsh)
3. Install zsh-autosuggest (following https://github.com/zsh-users/zsh-autosuggestions/blob/master/INSTALL.md#oh-my-zsh)

> How did you install powerlevel10k?

When there was a problem, I did above order differently. Instead of **1-2-3**, I did **1-3-2**. Installed zsh-autosuggest before powerlevel10k. That might somehow have caused the issue.


> What does your prompt look like in this case? The screenshot you posted is cut-off right before the prompt line. Can you show that line?

My prompt was looking like configurations were defaulted. There were to-be-diamond and to-be-lock symbols which I saw as "??". I cannot put it now, because I re-installed it and now it works fine.

> My guess is that your ~/.zshrc either doesn't get sourced or it exits early without reaching the end.
That makes sense to me.

> Could you attach your ~/.zshrc.
I re-installed and now working fine. I didn't keep the previous version unfortunately.
